### DISCUSSION TITLE
QOL: Right-clicking on the mods button should deselect all enabled mods.

### DISCUSSION THREAD

**Discussion by GabuTheDev** [ppy/osu#28301] [Ideas] [answered]
The random beatmap button has similar actions, click to random, right-click to rewind.
Same should apply for the mods button, click to select mods, right-click to quickly deselect.

**Comment by peppy** [comment]
I think this is a good idea, but let's wait for the new song select implementation to make it happen

cc @frenzibyte

**Comment by rodrigopina360** [comment]
In lazer you do have a keybind to deselect all mods, which although for now you can't assign to right click, you could atleast conceivably use it somewhere else until your suggestion is implemented.

**Reply by rodrigopina360** [reply]
Actually I just checked and it's not really appearing to work, not sure if it's something known, or an issue on my end, but here's the keybind name to check.
<img width="345" alt="image" src="https://github.com/ppy/osu/assets/41023844/e7fa9be7-b74d-474e-845e-413ee2ce8ba9">

**Comment by GabuTheDev** [comment]
Since song select v2 is closer than ever, I believe this should get some attention.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/33159
